examples/dockerfiles/README.md

```diff
@@ -0,0 +1,28 @@
+# Instructions for using the Dockerfiles
+
+To build these dockerfiles go to the Gluon-TS root directory and run:
+
+```bash 
+docker build . -f examples/dockerfiles/<the dockerfile>
+```
+
+To run the built image run:
+
+```bash
+docker run <image_id> <shell_params>
+```
+
+For more information about the shell and the available params, see the [shell documentation](https://github.com/awslabs/gluon-ts/tree/master/src/gluonts/shell) or run:
+
+```bash
+docker run <image_id> --help
+```
+
+## How to choose between the dockerfiles
+
+* Dockerfile
+    - This is for running Gluon-TS algorithms on a machine using only the CPU. This image is the most basic one and allows the use of most of the gluonts forecasters.
```

Comment:
I think we can abbreviate this by quite a bit, e.g.

```suggestion
    - Gluon-TS models using CPU.
```

Reply:
Oki